Error Occurred During Docker Build

### Self Checks

- [X] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Dify version

0.7.2

### Cloud or Self Hosted

Self Hosted (Docker)

### Steps to reproduce

- make build-all
- make build-api

<details>

<summary>docker build log</summary>

```docker
$ make build-api
Building API Docker image: langgenius/dify-api:latest...
docker build -t langgenius/dify-api:latest ./api

[+] Building 11.8s (10/17)                                                                                                  docker:default
 => [internal] load build definition from Dockerfile                                                                                  0.0s
 => => transferring dockerfile: 2.47kB                                                                                                0.0s
 => [internal] load metadata for docker.io/library/python:3.10-slim-bookworm                                                          0.6s
 => [internal] load .dockerignore                                                                                                     0.0s
 => => transferring context: 125B                                                                                                     0.0s
 => [base 1/3] FROM docker.io/library/python:3.10-slim-bookworm@sha256:8666a639a54acc810408e505e2c6b46b50834385701675ee177f578b3d2fd  0.0s
 => [internal] load build context                                                                                                     0.2s
 => => transferring context: 225.14kB                                                                                                 0.2s
 => CACHED [base 2/3] WORKDIR /app/api                                                                                                0.0s
 => CACHED [base 3/3] RUN pip install --no-cache-dir poetry==1.8.3                                                                    0.0s
 => CANCELED [packages 1/3] RUN apt-get update     && apt-get install -y --no-install-recommends gcc g++ libc-dev libffi-dev libgmp  11.2s
 => CACHED [production 1/7] WORKDIR /app/api                                                                                          0.0s
 => ERROR [production 2/7] RUN apt-get update     && apt-get install -y --no-install-recommends curl nodejs libgmp-dev libmpfr-dev   11.2s
------
 > [production 2/7] RUN apt-get update     && apt-get install -y --no-install-recommends curl nodejs libgmp-dev libmpfr-dev libmpc-dev     && echo "deb http://deb.debian.org/debian testing main" > /etc/apt/sources.list     && apt-get update     && apt-get install -y --no-install-recommends zlib1g=1:1.3.dfsg+really1.3.1-1 expat=2.6.2-1 libldap-2.5-0=2.5.18+dfsg-2 perl=5.38.2-5 libsqlite3-0=3.46.0-1     && apt-get autoremove -y     && rm -rf /var/lib/apt/lists/*:
0.331 Get:1 http://deb.debian.org/debian bookworm InRelease [151 kB]
0.346 Get:2 http://deb.debian.org/debian bookworm-updates InRelease [55.4 kB]
0.351 Get:3 http://deb.debian.org/debian-security bookworm-security InRelease [48.0 kB]
0.404 Get:4 http://deb.debian.org/debian bookworm/main amd64 Packages [8788 kB]
0.597 Get:5 http://deb.debian.org/debian bookworm-updates/main amd64 Packages [13.8 kB]
0.598 Get:6 http://deb.debian.org/debian-security bookworm-security/main amd64 Packages [179 kB]
1.469 Fetched 9235 kB in 1s (8028 kB/s)
1.469 Reading package lists...
1.938 Reading package lists...
2.407 Building dependency tree...
2.513 Reading state information...
2.655 The following additional packages will be installed:
2.655   libbrotli1 libc-ares2 libcurl4 libgmpxx4ldbl libicu72 libldap-2.5-0 libmpc3
2.655   libmpfr6 libnghttp2-14 libnode108 libpsl5 librtmp1 libsasl2-2
2.655   libsasl2-modules-db libssh2-1 libuv1 node-acorn node-busboy
2.656   node-cjs-module-lexer node-undici node-xtend
2.657 Suggested packages:
2.657   gmp-doc libgmp10-doc libmpfr-doc npm
2.657 Recommended packages:
2.657   libldap-common publicsuffix libsasl2-modules nodejs-doc
2.802 The following NEW packages will be installed:
2.803   curl libbrotli1 libc-ares2 libcurl4 libgmp-dev libgmpxx4ldbl libicu72
2.803   libldap-2.5-0 libmpc-dev libmpc3 libmpfr-dev libmpfr6 libnghttp2-14
2.803   libnode108 libpsl5 librtmp1 libsasl2-2 libsasl2-modules-db libssh2-1 libuv1
2.803   node-acorn node-busboy node-cjs-module-lexer node-undici node-xtend nodejs
2.828 0 upgraded, 26 newly installed, 0 to remove and 0 not upgraded.
2.828 Need to get 24.6 MB of archives.
2.828 After this operation, 101 MB of additional disk space will be used.
2.828 Get:1 http://deb.debian.org/debian bookworm/main amd64 libbrotli1 amd64 1.0.9-2+b6 [275 kB]
2.842 Get:2 http://deb.debian.org/debian bookworm/main amd64 libsasl2-modules-db amd64 2.1.28+dfsg-10 [20.3 kB]
2.843 Get:3 http://deb.debian.org/debian bookworm/main amd64 libsasl2-2 amd64 2.1.28+dfsg-10 [59.7 kB]
2.844 Get:4 http://deb.debian.org/debian bookworm/main amd64 libldap-2.5-0 amd64 2.5.13+dfsg-5 [183 kB]
2.846 Get:5 http://deb.debian.org/debian bookworm/main amd64 libnghttp2-14 amd64 1.52.0-1+deb12u1 [72.4 kB]
2.847 Get:6 http://deb.debian.org/debian bookworm/main amd64 libpsl5 amd64 0.21.2-1 [58.7 kB]
2.848 Get:7 http://deb.debian.org/debian bookworm/main amd64 librtmp1 amd64 2.4+20151223.gitfa8646d.1-2+b2 [60.8 kB]
2.849 Get:8 http://deb.debian.org/debian bookworm/main amd64 libssh2-1 amd64 1.10.0-3+b1 [179 kB]
2.851 Get:9 http://deb.debian.org/debian bookworm/main amd64 libcurl4 amd64 7.88.1-10+deb12u6 [390 kB]
2.860 Get:10 http://deb.debian.org/debian bookworm/main amd64 curl amd64 7.88.1-10+deb12u6 [314 kB]
2.863 Get:11 http://deb.debian.org/debian bookworm/main amd64 libc-ares2 amd64 1.18.1-3 [102 kB]
2.864 Get:12 http://deb.debian.org/debian bookworm/main amd64 libgmpxx4ldbl amd64 2:6.2.1+dfsg1-1.1 [338 kB]
2.867 Get:13 http://deb.debian.org/debian bookworm/main amd64 libgmp-dev amd64 2:6.2.1+dfsg1-1.1 [641 kB]
2.875 Get:14 http://deb.debian.org/debian bookworm/main amd64 libicu72 amd64 72.1-3 [9376 kB]
3.033 Get:15 http://deb.debian.org/debian bookworm/main amd64 libmpfr6 amd64 4.2.0-1 [701 kB]
3.045 Get:16 http://deb.debian.org/debian bookworm/main amd64 libmpc3 amd64 1.3.1-1 [51.5 kB]
3.046 Get:17 http://deb.debian.org/debian bookworm/main amd64 libmpfr-dev amd64 4.2.0-1 [259 kB]
3.051 Get:18 http://deb.debian.org/debian bookworm/main amd64 libmpc-dev amd64 1.3.1-1 [63.9 kB]
3.052 Get:19 http://deb.debian.org/debian bookworm/main amd64 libuv1 amd64 1.44.2-1+deb12u1 [136 kB]
3.055 Get:20 http://deb.debian.org/debian bookworm/main amd64 node-xtend all 4.0.2-3 [3932 B]
3.056 Get:21 http://deb.debian.org/debian bookworm/main amd64 nodejs amd64 18.19.0+dfsg-6~deb12u2 [319 kB]
3.061 Get:22 http://deb.debian.org/debian bookworm/main amd64 node-acorn all 8.8.1+ds+~cs25.17.7-2 [128 kB]
3.063 Get:23 http://deb.debian.org/debian bookworm/main amd64 node-cjs-module-lexer all 1.2.2+dfsg-5 [30.1 kB]
3.064 Get:24 http://deb.debian.org/debian bookworm/main amd64 node-busboy all 1.6.0+~cs2.6.0-2 [16.9 kB]
3.064 Get:25 http://deb.debian.org/debian bookworm/main amd64 node-undici all 5.15.0+dfsg1+~cs20.10.9.3-1+deb12u4 [285 kB]
3.069 Get:26 http://deb.debian.org/debian bookworm/main amd64 libnode108 amd64 18.19.0+dfsg-6~deb12u2 [10.5 MB]
3.376 debconf: delaying package configuration, since apt-utils is not installed
3.399 Fetched 24.6 MB in 0s (52.8 MB/s)
3.420 Selecting previously unselected package libbrotli1:amd64.
(Reading database ... 6696 files and directories currently installed.)
3.425 Preparing to unpack .../00-libbrotli1_1.0.9-2+b6_amd64.deb ...
3.431 Unpacking libbrotli1:amd64 (1.0.9-2+b6) ...
3.474 Selecting previously unselected package libsasl2-modules-db:amd64.
3.475 Preparing to unpack .../01-libsasl2-modules-db_2.1.28+dfsg-10_amd64.deb ...
3.478 Unpacking libsasl2-modules-db:amd64 (2.1.28+dfsg-10) ...
3.507 Selecting previously unselected package libsasl2-2:amd64.
3.508 Preparing to unpack .../02-libsasl2-2_2.1.28+dfsg-10_amd64.deb ...
3.510 Unpacking libsasl2-2:amd64 (2.1.28+dfsg-10) ...
3.541 Selecting previously unselected package libldap-2.5-0:amd64.
3.542 Preparing to unpack .../03-libldap-2.5-0_2.5.13+dfsg-5_amd64.deb ...
3.546 Unpacking libldap-2.5-0:amd64 (2.5.13+dfsg-5) ...
3.587 Selecting previously unselected package libnghttp2-14:amd64.
3.588 Preparing to unpack .../04-libnghttp2-14_1.52.0-1+deb12u1_amd64.deb ...
3.591 Unpacking libnghttp2-14:amd64 (1.52.0-1+deb12u1) ...
3.622 Selecting previously unselected package libpsl5:amd64.
3.624 Preparing to unpack .../05-libpsl5_0.21.2-1_amd64.deb ...
3.627 Unpacking libpsl5:amd64 (0.21.2-1) ...
3.658 Selecting previously unselected package librtmp1:amd64.
3.659 Preparing to unpack .../06-librtmp1_2.4+20151223.gitfa8646d.1-2+b2_amd64.deb ...
3.662 Unpacking librtmp1:amd64 (2.4+20151223.gitfa8646d.1-2+b2) ...
3.695 Selecting previously unselected package libssh2-1:amd64.
3.696 Preparing to unpack .../07-libssh2-1_1.10.0-3+b1_amd64.deb ...
3.699 Unpacking libssh2-1:amd64 (1.10.0-3+b1) ...
3.735 Selecting previously unselected package libcurl4:amd64.
3.736 Preparing to unpack .../08-libcurl4_7.88.1-10+deb12u6_amd64.deb ...
3.739 Unpacking libcurl4:amd64 (7.88.1-10+deb12u6) ...
3.783 Selecting previously unselected package curl.
3.784 Preparing to unpack .../09-curl_7.88.1-10+deb12u6_amd64.deb ...
3.787 Unpacking curl (7.88.1-10+deb12u6) ...
3.829 Selecting previously unselected package libc-ares2:amd64.
3.830 Preparing to unpack .../10-libc-ares2_1.18.1-3_amd64.deb ...
3.832 Unpacking libc-ares2:amd64 (1.18.1-3) ...
3.866 Selecting previously unselected package libgmpxx4ldbl:amd64.
3.867 Preparing to unpack .../11-libgmpxx4ldbl_2%3a6.2.1+dfsg1-1.1_amd64.deb ...
3.870 Unpacking libgmpxx4ldbl:amd64 (2:6.2.1+dfsg1-1.1) ...
3.901 Selecting previously unselected package libgmp-dev:amd64.
3.903 Preparing to unpack .../12-libgmp-dev_2%3a6.2.1+dfsg1-1.1_amd64.deb ...
3.906 Unpacking libgmp-dev:amd64 (2:6.2.1+dfsg1-1.1) ...
3.958 Selecting previously unselected package libicu72:amd64.
3.959 Preparing to unpack .../13-libicu72_72.1-3_amd64.deb ...
3.961 Unpacking libicu72:amd64 (72.1-3) ...
4.446 Selecting previously unselected package libmpfr6:amd64.
4.447 Preparing to unpack .../14-libmpfr6_4.2.0-1_amd64.deb ...
4.450 Unpacking libmpfr6:amd64 (4.2.0-1) ...
4.502 Selecting previously unselected package libmpc3:amd64.
4.503 Preparing to unpack .../15-libmpc3_1.3.1-1_amd64.deb ...
4.507 Unpacking libmpc3:amd64 (1.3.1-1) ...
4.534 Selecting previously unselected package libmpfr-dev:amd64.
4.535 Preparing to unpack .../16-libmpfr-dev_4.2.0-1_amd64.deb ...
4.538 Unpacking libmpfr-dev:amd64 (4.2.0-1) ...
4.579 Selecting previously unselected package libmpc-dev:amd64.
4.580 Preparing to unpack .../17-libmpc-dev_1.3.1-1_amd64.deb ...
4.582 Unpacking libmpc-dev:amd64 (1.3.1-1) ...
4.616 Selecting previously unselected package libuv1:amd64.
4.617 Preparing to unpack .../18-libuv1_1.44.2-1+deb12u1_amd64.deb ...
4.620 Unpacking libuv1:amd64 (1.44.2-1+deb12u1) ...
4.651 Selecting previously unselected package node-xtend.
4.652 Preparing to unpack .../19-node-xtend_4.0.2-3_all.deb ...
4.655 Unpacking node-xtend (4.0.2-3) ...
4.682 Selecting previously unselected package nodejs.
4.683 Preparing to unpack .../20-nodejs_18.19.0+dfsg-6~deb12u2_amd64.deb ...
4.686 Unpacking nodejs (18.19.0+dfsg-6~deb12u2) ...
4.744 Selecting previously unselected package node-acorn.
4.745 Preparing to unpack .../21-node-acorn_8.8.1+ds+~cs25.17.7-2_all.deb ...
4.748 Unpacking node-acorn (8.8.1+ds+~cs25.17.7-2) ...
4.791 Selecting previously unselected package node-cjs-module-lexer.
4.792 Preparing to unpack .../22-node-cjs-module-lexer_1.2.2+dfsg-5_all.deb ...
4.796 Unpacking node-cjs-module-lexer (1.2.2+dfsg-5) ...
4.821 Selecting previously unselected package node-busboy.
4.822 Preparing to unpack .../23-node-busboy_1.6.0+~cs2.6.0-2_all.deb ...
4.826 Unpacking node-busboy (1.6.0+~cs2.6.0-2) ...
4.854 Selecting previously unselected package node-undici.
4.856 Preparing to unpack .../24-node-undici_5.15.0+dfsg1+~cs20.10.9.3-1+deb12u4_all.deb ...
4.863 Unpacking node-undici (5.15.0+dfsg1+~cs20.10.9.3-1+deb12u4) ...
4.914 Selecting previously unselected package libnode108:amd64.
4.915 Preparing to unpack .../25-libnode108_18.19.0+dfsg-6~deb12u2_amd64.deb ...
4.918 Unpacking libnode108:amd64 (18.19.0+dfsg-6~deb12u2) ...
5.424 Setting up libpsl5:amd64 (0.21.2-1) ...
5.434 Setting up libicu72:amd64 (72.1-3) ...
5.440 Setting up libbrotli1:amd64 (1.0.9-2+b6) ...
5.449 Setting up libnghttp2-14:amd64 (1.52.0-1+deb12u1) ...
5.459 Setting up node-cjs-module-lexer (1.2.2+dfsg-5) ...
5.465 Setting up libc-ares2:amd64 (1.18.1-3) ...
5.472 Setting up libsasl2-modules-db:amd64 (2.1.28+dfsg-10) ...
5.481 Setting up libgmpxx4ldbl:amd64 (2:6.2.1+dfsg1-1.1) ...
5.489 Setting up libuv1:amd64 (1.44.2-1+deb12u1) ...
5.496 Setting up libmpfr6:amd64 (4.2.0-1) ...
5.503 Setting up librtmp1:amd64 (2.4+20151223.gitfa8646d.1-2+b2) ...
5.511 Setting up libmpc3:amd64 (1.3.1-1) ...
5.518 Setting up libsasl2-2:amd64 (2.1.28+dfsg-10) ...
5.525 Setting up libssh2-1:amd64 (1.10.0-3+b1) ...
5.533 Setting up node-xtend (4.0.2-3) ...
5.540 Setting up node-busboy (1.6.0+~cs2.6.0-2) ...
5.547 Setting up libgmp-dev:amd64 (2:6.2.1+dfsg1-1.1) ...
5.554 Setting up libmpfr-dev:amd64 (4.2.0-1) ...
5.561 Setting up libldap-2.5-0:amd64 (2.5.13+dfsg-5) ...
5.567 Setting up libmpc-dev:amd64 (1.3.1-1) ...
5.573 Setting up node-undici (5.15.0+dfsg1+~cs20.10.9.3-1+deb12u4) ...
5.586 Setting up libcurl4:amd64 (7.88.1-10+deb12u6) ...
5.593 Setting up curl (7.88.1-10+deb12u6) ...
5.603 Setting up node-acorn (8.8.1+ds+~cs25.17.7-2) ...
5.609 Setting up libnode108:amd64 (18.19.0+dfsg-6~deb12u2) ...
5.616 Setting up nodejs (18.19.0+dfsg-6~deb12u2) ...
5.624 update-alternatives: using /usr/bin/nodejs to provide /usr/bin/js (js) in auto mode
5.624 update-alternatives: warning: skip creation of /usr/share/man/man1/js.1.gz because associated file /usr/share/man/man1/nodejs.1.gz (of link group js) doesn't exist
5.628 Processing triggers for libc-bin (2.36-9+deb12u7) ...
5.913 Get:1 http://deb.debian.org/debian testing InRelease [169 kB]
5.927 Hit:2 http://deb.debian.org/debian bookworm InRelease
5.929 Hit:3 http://deb.debian.org/debian bookworm-updates InRelease
5.933 Hit:4 http://deb.debian.org/debian-security bookworm-security InRelease
6.498 Get:5 http://deb.debian.org/debian testing/main amd64 Packages [9190 kB]
8.976 Fetched 9359 kB in 3s (2834 kB/s)
8.976 Reading package lists...
9.948 Reading package lists...
10.87 Building dependency tree...
11.06 Reading state information...
11.07 Package libldap-2.5-0 is not available, but is referred to by another package.
11.07 This may mean that the package is missing, has been obsoleted, or
11.07 is only available from another source
11.07
```
</details>

~~~
11.07 E: Version '2.5.18+dfsg-2' for 'libldap-2.5-0' was not found
------
Dockerfile:51
--------------------
  50 |
  51 | >>> RUN apt-get update \
  52 | >>>     && apt-get install -y --no-install-recommends curl nodejs libgmp-dev libmpfr-dev libmpc-dev \
  53 | >>>     # if you located in China, you can use aliyun mirror to speed up
  54 | >>>     # && echo "deb http://mirrors.aliyun.com/debian testing main" > /etc/apt/sources.list \
  55 | >>>     && echo "deb http://deb.debian.org/debian testing main" > /etc/apt/sources.list \
  56 | >>>     && apt-get update \
  57 | >>>     # For Security
  58 | >>>     && apt-get install -y --no-install-recommends zlib1g=1:1.3.dfsg+really1.3.1-1 expat=2.6.2-1 libldap-2.5-0=2.5.18+dfsg-2 perl=5.38.2-5 libsqlite3-0=3.46.0-1 \
  59 | >>>     && apt-get autoremove -y \
  60 | >>>     && rm -rf /var/lib/apt/lists/*
  61 |
--------------------
ERROR: failed to solve: process "/bin/sh -c apt-get update     && apt-get install -y --no-install-recommends curl nodejs libgmp-dev libmpfr-dev libmpc-dev     && echo \"deb http://deb.debian.org/debian testing main\" > /etc/apt/sources.list     && apt-get update     && apt-get install -y --no-install-recommends zlib1g=1:1.3.dfsg+really1.3.1-1 expat=2.6.2-1 libldap-2.5-0=2.5.18+dfsg-2 perl=5.38.2-5 libsqlite3-0=3.46.0-1     && apt-get autoremove -y     && rm -rf /var/lib/apt/lists/*" did not complete successfully: exit code: 100
make: *** [Makefile:15: build-api] Error 1
~~~


### ✔️ Expected Behavior

- libldap-2.5-0=2.5.18+dfsg-`2` to libldap-2.5-0=2.5.18+dfsg-`3`

~~~
 55 | >>>     && echo "deb http://deb.debian.org/debian testing main" > /etc/apt/sources.list \
~~~

![image](https://github.com/user-attachments/assets/2aae4f27-2962-45d0-97c3-db29b7d8f337)

ref.
https://packages.debian.org/search?lang=en&suite=all&arch=any&searchon=names&keywords=libldap-2.5

### ❌ Actual Behavior

# Solution

## update Dockerfile

~~~diff
$ git diff
diff --git a/api/Dockerfile b/api/Dockerfile
index cca64886..82b89ad7 100644
--- a/api/Dockerfile
+++ b/api/Dockerfile
@@ -55,7 +55,7 @@ RUN apt-get update \
     && echo "deb http://deb.debian.org/debian testing main" > /etc/apt/sources.list \
     && apt-get update \
     # For Security
-    && apt-get install -y --no-install-recommends zlib1g=1:1.3.dfsg+really1.3.1-1 expat=2.6.2-1 libldap-2.5-0=2.5.18+dfsg-2 perl=5.38.2-5 libsqlite3-0=3.46.0-1 \
+    && apt-get install -y --no-install-recommends zlib1g=1:1.3.dfsg+really1.3.1-1 expat=2.6.2-1 libldap-2.5-0=2.5.18+dfsg-3 perl=5.38.2-5 libsqlite3-0=3.46.0-1 \
     && apt-get autoremove -y \
     && rm -rf /var/lib/apt/lists/*

~~~

<details open>

<summary>docker build log</summary>

~~~
$ make build-api
Building API Docker image: langgenius/dify-api:latest...
docker build -t langgenius/dify-api:latest ./api
[+] Building 110.1s (18/18) FINISHED                                                                                        docker:default
 => [internal] load build definition from Dockerfile                                                                                  0.0s
 => => transferring dockerfile: 2.47kB                                                                                                0.0s
 => [internal] load metadata for docker.io/library/python:3.10-slim-bookworm                                                          1.1s
 => [internal] load .dockerignore                                                                                                     0.0s
 => => transferring context: 125B                                                                                                     0.0s
 => [internal] load build context                                                                                                     0.2s
 => => transferring context: 227.58kB                                                                                                 0.2s
 => [base 1/3] FROM docker.io/library/python:3.10-slim-bookworm@sha256:8666a639a54acc810408e505e2c6b46b50834385701675ee177f578b3d2fd  0.0s
 => CACHED [production 1/7] WORKDIR /app/api                                                                                          0.0s
 => CACHED [base 2/3] WORKDIR /app/api                                                                                                0.0s
 => CACHED [base 3/3] RUN pip install --no-cache-dir poetry==1.8.3                                                                    0.0s
 => [production 2/7] RUN apt-get update     && apt-get install -y --no-install-recommends curl nodejs libgmp-dev libmpfr-dev libmpc  18.8s
 => [packages 1/3] RUN apt-get update     && apt-get install -y --no-install-recommends gcc g++ libc-dev libffi-dev libgmp-dev libm  11.1s
 => [packages 2/3] COPY pyproject.toml poetry.lock ./                                                                                 0.0s
 => [packages 3/3] RUN poetry install --sync --no-cache --no-root                                                                    59.9s
 => [production 3/7] COPY --from=packages /app/api/.venv /app/api/.venv                                                               7.8s
 => [production 4/7] RUN python -c "import nltk; nltk.download('punkt'); nltk.download('averaged_perceptron_tagger')"                 3.1s
 => [production 5/7] COPY . /app/api/                                                                                                 0.3s
 => [production 6/7] COPY docker/entrypoint.sh /entrypoint.sh                                                                         0.0s
 => [production 7/7] RUN chmod +x /entrypoint.sh                                                                                      0.2s
 => exporting to image                                                                                                               14.0s
 => => exporting layers                                                                                                              14.0s
 => => writing image sha256:617c8273f6054fc64120c4f2fe1baf3174cbc0d111cfadfce6c656893e9bb748                                          0.0s
 => => naming to docker.io/langgenius/dify-api:latest                                                                                 0.0s
API Docker image built successfully: langgenius/dify-api:latest
~~~

</details>

# Reason

During the CI build process, versions are locked, and it seems that the issue went unnoticed due to caching.

https://github.com/langgenius/dify/actions/runs/10577993158/job/29307147730#step:7:378

--- a/api/Dockerfile
+++ b/api/Dockerfile
@@ -55,7 +55,7 @@ RUN apt-get update \
     && echo "deb http://deb.debian.org/debian testing main" > /etc/apt/sources.list \
     && apt-get update \
     # For Security
-    && apt-get install -y --no-install-recommends zlib1g=1:1.3.dfsg+really1.3.1-1 expat=2.6.2-1 libldap-2.5-0=2.5.18+dfsg-2 perl=5.38.2-5 libsqlite3-0=3.46.0-1 \
+    && apt-get install -y --no-install-recommends zlib1g=1:1.3.dfsg+really1.3.1-1 expat=2.6.2-1 libldap-2.5-0=2.5.18+dfsg-3 perl=5.38.2-5 libsqlite3-0=3.46.0-1 \
     && apt-get autoremove -y \
     && rm -rf /var/lib/apt/lists/*
 

